frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -576,13 +666,35 @@ export function PollWorkerScreen({
             </Prose>
           ) : (
             <Prose>
-              <p>
-                <Button onPress={() => setIsShowingVxScanPollsOpenModal(true)}>
-                  {isPollsOpen
-                    ? `Close Polls for ${precinctName}`
-                    : `Open Polls for ${precinctName}`}
-                </Button>
-              </p>
+              {pollsState === 'polls_closed_initial' && (
```

Comment:
Don't you have helper functions that map from the state to the transition you want? Could you use that here or a similar helper instead of having to have all of these checks?

Reply:
I could have a similar helper, yes, I'll do that.

Reply:
Done